template-only-bin/update-template.sh

```diff
@@ -6,23 +6,30 @@ set -euo pipefail
 
 SCRIPT_DIR=$(dirname $0)
 
-echo "Install template"
+# echo "Install template"
 $SCRIPT_DIR/install-template.sh
 
-# Restore project files with project-specific configuration that was defined as part of project setup.
-# This includes the terraform backend configuration blocks and the project-config module
-# Also restore project files that had lines that were commented out in the template, such as Makefile
-# and cd.yml workflow
-# Updates in any of these files need to be manually applied to the projects
-echo "Restore modified project files"
-git checkout HEAD -- \
-  .dockleconfig \
-  .github/workflows/build-and-publish.yml \
-  .github/workflows/cd.yml \
-  .github/workflows/ci-infra.yml \
-  .github/workflows/database-migrations.yml \
-  .grype.yml \
-  .hadolint.yaml \
-  .trivyignore \
-  infra/project-config/main.tf \
-  infra/app/app-config/main.tf
+
+# This is the last commit from template-infra that the project had access to.
+PROJECT_VERSION=$(cat bin/template-version.txt)
+# Current HEAD of the template repository
+TEMPLATE_VERSION=$(git rev-parse HEAD)
```

Comment:
This isn’t correct. This script is run from the project’s root directory (see comment at top of file) so git rev-parse HEAD will return the project’s current hash not the template’s